src/ClimaAtmos/Parameterizations/TurbulenceConvection/src/TurbulenceConvection.jl

```diff
@@ -0,0 +1,8 @@
+module TurbulenceConvection
+
+include("Grids/Grids.jl")
```

Comment:
These files probably don't need to be in subdirectories

Reply:
I'd actually prefer to keep `Grids` in its own subdirectory since there are several more components coming and I'd like to not crowd `TurbulenceConvection/src/`.

I can combine `GridOperators` so that there are fewer subdirectories. Can we wait to collapse the subdirectories?

Reply:
yeah, just noting that complicated directory structures can get confusing.